pkg/cmd/artefactor.go

```diff
@@ -1,6 +1,7 @@
 package cmd
```

Comment:
Rename repository to `artefactor` (or all code references to `artifactor`), for consistency.

Reply:
README can also do with an update including description, example use cases and how-to etc.

Reply:
All done - it's UK though so artefactor.